api/src/main/java/org/openmrs/api/context/ServiceContext.java

```diff
@@ -143,10 +143,10 @@ public static ServiceContext getInstance() {
 	 */
 	public static void destroyInstance() {
 		if (ServiceContextHolder.instance != null && ServiceContextHolder.instance.services != null) {
-			if (log.isDebugEnabled()) {
+				
```

Comment:
please remove the indentation since you have removed an if block we do not need the extra tab